### PR TITLE
SNow v2 fix empty credentials error

### PR DESCRIPTION
**Pull request by darkushin**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/29126

## Description
Fixing an issue where an empty username/password in the instance configurations would result in a non-indicative error.

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

File: Packs/ServiceNow/ReleaseNotes/1_3_3.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### ServiceNow v2
+- Fixed a bug where empty username or password in the instance configurations would result in an error.
```
Comment (bakatzir) [inline Packs/ServiceNow/ReleaseNotes/1_3_3.md:4]:
```suggestion
- Fixed a bug issue empty username or password in the instance configuration resulted in a non informative error.
```

**Review by bakatzir — COMMENTED** [review]
How does the error look now? attach a screenshot please.
also, add an rn regarding the dockerimage update.

**Comment by darkushin** [comment]
@bakatzir 
> How does the error look now? attach a screenshot please.
> also, add an rn regarding the dockerimage update.

image
The error message is a bit long, but it indicates the correct reason for the error in the beginning: `User Not Authenticated.. Required to provide Auth information.`

**Review by bakatzir — APPROVED** [review]
legit. and clearer than before.

**Pull request merged**